Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -174,27 +175,30 @@ public void StageTransactions()
             };
             Assert.Empty(txs.Where(tx => _fx.Store.ContainsTransaction(tx.Id)));
 
-            _blockChain.StageTransactions(txs.ToImmutableHashSet());
+            StageTransactions(txs);
+
             Assert.Equal(
                 txs,
                 txs.Where(tx => _fx.Store.ContainsTransaction(tx.Id)).ToHashSet()
             );
         }
 
         [Fact]
-        public void UnstageTransactions()
+        public void UnstageTransaction()
         {
             Assert.Empty(_blockChain.GetStagedTransactionIds());
-            var txs = new HashSet<Transaction<DumbAction>>()
-            {
-                _fx.Transaction1,
-                _fx.Transaction2,
-            };
-            _blockChain.StageTransactions(txs.ToImmutableHashSet());
-            HashSet<TxId> txIds = txs.Select(tx => tx.Id).ToHashSet();
+
+            _blockChain.StageTransaction(_fx.Transaction1);
+            _blockChain.StageTransaction(_fx.Transaction2);
```

Comment:
Is it intended to not use `StageTransactions`?

Reply:
There is no special reason. I'll fix it to use.